synchronization/supervized/supervisor.go

```diff
@@ -0,0 +1,72 @@
+package supervized
+
+import (
+	"context"
+	"fmt"
+	"github.com/orbs-network/orbs-network-go/instrumentation/log"
+	"github.com/pkg/errors"
+	"runtime"
+	"strings"
+)
+
+type Errorer interface {
+	Error(message string, fields ...*log.Field)
+}
+
+func OneOff(logger Errorer, f func()) {
```

Comment:
Maybe rename `OneOff` to `ShortLived`?

Reply:
that's not the semantic; rather - the OneOff is just a single run, while the LongLived should not be allowed to return until context is closed.

Reply:
`ShortLived` can have the same semantics. `LongLived` means needs to destroy the context to end, `ShortLived` doesn't need a context at all because it terminates quickly